src/main/kotlin/com/sourcegraph/cody/auth/SelectOneOfTheAccountsAsActive.kt

```diff
@@ -3,24 +3,19 @@ package com.sourcegraph.cody.auth
 import com.intellij.openapi.project.Project
 import com.sourcegraph.cody.CodyToolWindowContent
 import com.sourcegraph.cody.config.CodyAuthenticationManager
-import com.sourcegraph.cody.config.CodyProjectActiveAccountHolder
 import com.sourcegraph.cody.config.getFirstAccountOrNull
 import com.sourcegraph.cody.initialization.Activity
 
 class SelectOneOfTheAccountsAsActive : Activity {
```

Comment:
Cannot we now move that whole code to `CodyAuthenticationManager::loadState`?

Reply:
good point, lemme test it...

Reply:
needed to override noStateLoaded to get it working without the active account set initially (loadState executes only if there is a state)

docs:
https://plugins.jetbrains.com/docs/intellij/persisting-state-of-components.html#customizing-the-xml-format-of-persisted-values